requirement-generator/requirement-satisfiable-course-generator.ts

```diff
@@ -0,0 +1,75 @@
+import { readFileSync, writeFileSync } from 'fs';
+import { Course } from './types';
+import {
+  StrictFulfilledByType,
+  MajorRequirement,
+  CollegeRequirement,
+  RequirementsJson
+} from '../src/requirements/types';
+
+type FilteredCourse = Pick<Course, 'subject' | 'catalogNbr'>;
+
+const filteredAllCourses: readonly FilteredCourse[] = JSON.parse(
+  readFileSync('requirement-generator/filtered-all-courses.json').toString()
+);
+const requirementJson: RequirementsJson<StrictFulfilledByType> = JSON.parse(
+  readFileSync('src/requirements/reqs.json').toString()
+);
+
+const courseCanSatisfyCollegeRequirement = (
+  course: FilteredCourse,
+  majorRequirement: CollegeRequirement<StrictFulfilledByType>
+): boolean => Math.random() < 0.01;
+
+const courseCanSatisfyMajorRequirement = (
+  course: FilteredCourse,
+  majorRequirement: MajorRequirement<StrictFulfilledByType>
+): boolean => Math.random() < 0.01;
```

Comment:
Don't want to keep the randomized method for filtering courses on master for too long